### PR TITLE
treewide: xdg_utils -> xdg-utils

### PR DESCRIPTION
**Pull request by siraben**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/graphics/emulsion/default.nix [hunk 1]
```diff
@@ -15,7 +15,7 @@
 , libxcb
 , libxkbcommon
 , wayland
-, xdg_utils
+, xdg-utils
```
Comment (SuperSandro2000) [inline pkgs/applications/graphics/emulsion/default.nix:18]: Is that input unused?

File: pkgs/top-level/all-packages.nix [hunk 2]
```diff
@@ -25854,7 +25854,7 @@ in
 
   xdg-user-dirs = callPackage ../tools/X11/xdg-user-dirs { };
 
-  xdg_utils = callPackage ../tools/X11/xdg-utils {
+  xdg-utils = callPackage ../tools/X11/xdg-utils {
```
Comment (SuperSandro2000) [inline pkgs/top-level/all-packages.nix:25857]: I think we should add an alias in aliases.nix.

File: pkgs/applications/graphics/emulsion/default.nix (hunk 1 above)
Comment (siraben) [reply pkgs/applications/graphics/emulsion/default.nix:18]: It appears so. I will make a separate commit.

**Comment by jtojnar** [comment]
Thanks.

**Pull request merged**